package.json

```
@@ -57,6 +61,7 @@
   "bugs": "https://github.com/eslint/eslint/issues/",
   "dependencies": {
     "@eslint/eslintrc": "^1.4.1",
+    "@eslint/js": "^8.0.0",
```

```suggestion
    "@eslint/js": "8.33.0",
```

I think we should pin the version so that when we add a new rule it doesn't break users of `eslint:all`.

---

Makes sense to me.